math/mathcore/inc/VectorizedTMath.h

```diff
@@ -0,0 +1,765 @@
+#ifndef ROOT_VectorizedTMath
+#define ROOT_VectorizedTMath
+
+#include "Rtypes.h"
+#include "Math/Types.h"
+#include "TMath.h"
+
+#if defined(R__HAS_VECCORE) && defined(R__HAS_VC)
+
+namespace TMath {
+////////////////////////////////////////////////////////////////////////////////
+::ROOT::Double_v Log2(::ROOT::Double_v &x)
+{
+   return vecCore::math::Log2(x);
+}
+
+////////////////////////////////////////////////////////////////////////////////
+/// Calculate a Breit Wigner function with mean and gamma.
+::ROOT::Double_v BreitWigner(::ROOT::Double_v &x, Double_t mean=0, Double_t gamma=1)
+{
+   return (gamma / (gamma*gamma / 4.0f + (x-mean) * (x-mean)))/(2*Pi());
```

Comment:
You are mixing `float` and `double` here (i.e. `4.0f`). This creates unnecessary conversions that hurt performance. Also, please use `* 0.5 * M_1_PI` instead of `/ (2 * Pi())` to save a division.